packages/switch/src/component.tsx

```diff
@@ -5,8 +5,34 @@ import { switchToggle as ccSwitch } from '@warp-ds/css/component-classes';
 
 import { SwitchProps } from './props.js';
 
-export function Switch({ id, value, onClick, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, ...attrs }: SwitchProps) {
-  const switchFocus = 'focusable rounded-full';
+export function Switch({
+  id,
+  value,
+  disabled,
+  onClick,
+  'aria-label': ariaLabel,
+  'aria-labelledby': ariaLabelledBy,
+  ...attrs
+}: SwitchProps) {
+  const baseClasses = classNames([ccSwitch.base, disabled && ccSwitch.disabled]);
+
+  const trackClasses = classNames([
+    ccSwitch.track,
+    disabled && ccSwitch.trackDisabled,
+    value && !disabled ? ccSwitch.trackActive : ccSwitch.trackInactive,
```

Comment:
Should this condition look similar as you did in a respective [Vue PR](https://github.com/warp-ds/vue/pull/187#discussion_r1706656863)?
```suggestion
    disabled && ccSwitch.trackDisabled,
    !disabled && (value ? ccSwitch.trackActive : ccSwitch.trackInactive)
```

Reply:
Yes you're right! 😄  I'll change it! 🎉

Reply:
Updated in this commit: [fb41ad8](https://github.com/warp-ds/react/pull/270/commits/fb41ad85ab2fbdb3561e8692df4e89c07f63fcb0)